src/Assets/TestProjects/KitchenSink/TestApp/TestApp.csproj

```diff
@@ -25,6 +25,7 @@
     <NullabilityInfoContextSupport>false</NullabilityInfoContextSupport>
     <CustomResourceTypesSupport>false</CustomResourceTypesSupport>
     <UseSystemResourceKeys>true</UseSystemResourceKeys>
+    <DynamicCodeSupport>false</DynamicCodeSupport>
```

Comment:
This is a test project which is used in a lot of different tests, does it make sense to turn of dynamic code support for all the tests that use the project?

Reply:
I am just testing that the value flows through to the runtimeconfig.json. I can flip it to `true` to not affect other tests.